src/assessments/seo/KeywordDensityAssessment.js

```diff
@@ -68,20 +127,26 @@ class KeywordDensityAssessment extends Assessment {
 	 * @returns {AssessmentResult} The result of the assessment.
 	 */
 	getResult( paper, researcher, i18n ) {
-		const assessmentResult = new AssessmentResult();
+		// Get the environment variable.
+		this._hasMorphologicalForms = researcher.getData( "morphology" ) !== false && paper.getLocale() === "en_EN";
```

Comment:
This check shouldn't use `getLocale`. As it is, it will evaluate to `false` if the locale is `en_US` for example. Rather than checking the locale, it should check the language.